presto-main/src/main/java/io/prestosql/util/FinalizerService.java

```diff
@@ -105,16 +101,16 @@ private void processFinalizerQueue()
     private static class FinalizerReference
             extends PhantomReference<Object>
     {
-        private final Runnable cleanup;
-        private final AtomicBoolean executed = new AtomicBoolean();
+        final Runnable cleanup;
+        final AtomicBoolean executed = new AtomicBoolean();
 
-        public FinalizerReference(Object referent, ReferenceQueue<Object> queue, Runnable cleanup)
```

Comment:
I disagree with removing `public` from these. To me, `public` on a constructor or method indicates that it is logically part of the public API. The visibility of the class is controlled at the class level. Consider the scenario where we want to make this class public. With the members already marked as `public`, as all we need to do is change the class visibility. If we make them selectively package private, we now need to think about which ones should be made public as part of the API..